### PR TITLE
[release] bump 7.9 branch to 7.9.2-SNAPSHOT 

### PR DESCRIPTION
**Pull request by ygel**
[release] bump 7.9 branch to 7.9.2-SNAPSHOT 

See [New branching model](https://github.com/elastic/helm-charts/blob/master/BREAKING_CHANGES.md#new-branching-model) for more details.

### CODE REVIEW

File: apm-server/README.md [hunk 1]
```diff
@@ -8,13 +8,18 @@ The design and code is less mature than official GA features and is being
 provided as-is with no warranties. Alpha features are not subject to the support
 SLA of official GA features (see [supported configurations][] for more details).
 
+**Warning**: This branch is used for development, please use [7.9.1][] release
+for released version.
+
+
```
Comment (jmlrt) [inline apm-server/README.md:14]:
```suggestion
**Warning**: This branch is used for development, please use [7.9.1][] release for released version.
```

I was thinking about keeping this message on a single line now, this way we can drop it more easily with `sed -i -e "/This branch is used for development, please use .* release for released version\./d" */README.md` when bumping again to release version (see https://github.com/elastic/infra/blob/master/docs/releases/helm-charts.md#bump-the-release-branch-from-snapshot-to-release-version)

File: elasticsearch/README.md [hunk 2]
```diff
@@ -3,13 +3,18 @@
 This Helm chart is a lightweight way to configure and run our official
 [Elasticsearch Docker image][].
 
+**Warning**: This branch is used for development, please use [7.9.1][] release
+for released version.
+
+
```
Comment (jmlrt) [inline elasticsearch/README.md:9]:
```suggestion
**Warning**: This branch is used for development, please use [7.9.1][] release for released version.
```

File: filebeat/README.md [hunk 3]
```diff
@@ -3,13 +3,18 @@
 This Helm chart is a lightweight way to configure and run our official
 [Filebeat Docker image][].
 
+**Warning**: This branch is used for development, please use [7.9.1][] release
+for released version.
+
+
```
Comment (jmlrt) [inline filebeat/README.md:9]:
```suggestion
**Warning**: This branch is used for development, please use [7.9.1][] release for released version.
```

File: kibana/README.md [hunk 4]
```diff
@@ -3,13 +3,18 @@
 This Helm chart is a lightweight way to configure and run our official
 [Kibana Docker image][].
 
+**Warning**: This branch is used for development, please use [7.9.1][] release
+for released version.
+
+
```
Comment (jmlrt) [inline kibana/README.md:9]:
```suggestion
**Warning**: This branch is used for development, please use [7.9.1][] release for released version.
```

File: logstash/README.md [hunk 5]
```diff
@@ -8,13 +8,18 @@ The design and code is less mature than official GA features and is being
 provided as-is with no warranties. Alpha features are not subject to the support
 SLA of official GA features (see [supported configurations][] for more details).
 
+**Warning**: This branch is used for development, please use [7.9.1][] release
+for released version.
+
+
```
Comment (jmlrt) [inline logstash/README.md:14]:
```suggestion
**Warning**: This branch is used for development, please use [7.9.1][] release for released version.
```

File: metricbeat/README.md [hunk 6]
```diff
@@ -3,13 +3,18 @@
 This Helm chart is a lightweight way to configure and run our official
 [Metricbeat Docker image][].
 
+**Warning**: This branch is used for development, please use [7.9.1][] release
+for released version.
+
+
```
Comment (jmlrt) [inline metricbeat/README.md:9]:
```suggestion
**Warning**: This branch is used for development, please use [7.9.1][] release for released version.
```

**Review by jmlrt — COMMENTED** [review]
Just a proposal to make the warning notice single line so we can remove with a simple `sed` snippet next time (maybe we could also add it with a sed snippet).

The rest LGTM and pass the CI tests

**Comment by ygel** [comment]
Makes perfect sense, done!

**Pull request merged**